compositor_pipeline/src/audio_mixer/types.rs

```diff
@@ -70,6 +93,18 @@ impl AudioSamples {
     }
 }
 
+impl Debug for InputSamples {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
```

Comment:
There is a helper for struct debug implementations
https://github.com/membraneframework/video_compositor/blob/fa44592411ac7bd4d137ca40f6f690ff6196615a/compositor_render/src/types.rs#L24